Lib/typing.py

```diff
@@ -656,6 +657,8 @@ def _evaluate(self, globalns, localns, recursive_guard):
                 globalns = localns
             elif localns is None:
                 localns = globalns
+            if self.__forward_module__ is not None:
+                globalns = sys.modules[self.__forward_module__].__dict__
```

Comment:
It's possible for `self.__forward_module__` to not be in `sys.modules` and raise `KeyError`. Maybe using this style would be better? https://github.com/will-ca/cpython/blob/090bf13b927e0eada702110806c0276f265e9584/Lib/typing.py#L1704